src/index.js

```diff
@@ -1,9 +1,12 @@
 const express = require('express');
 const cors = require('cors');
 const routes = require('./routes');
+const db = require('./db');
 
 const app = express();
 
+db.migrate();
```

Comment:
is `db.migrate` synchronous operation? Do we need to wait for migration finish to start working with `log.txt`?